### PR TITLE
Add PeerDAS presets and configs.

### PR DESCRIPTION
**Pull request by jimmygchen**
## Issue Addressed

Part of #6072

## Proposed Changes

This PR pulls all presets and configs related changes from the `das` branch.

### CODE REVIEW

File: consensus/types/src/chain_spec.rs [hunk 1]
```diff
@@ -1426,6 +1477,10 @@ const fn default_max_request_blob_sidecars() -> u64 {
     768
 }
 
+const fn default_max_request_data_column_sidecars() -> u64 {
+    512
```
Comment (pawanjay176) [inline consensus/types/src/chain_spec.rs]: Spec says 16384 🤔 
https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L163

**Review by pawanjay176 — CHANGES_REQUESTED** [review]
Looks good except a config value

**Review by realbigsean — APPROVED** [review]
looks good other than pawan's comment

File: consensus/types/src/chain_spec.rs (hunk 1 above)
Comment (jimmygchen) [reply consensus/types/src/chain_spec.rs]: Good catch! Fixed

**Review by pawanjay176 — APPROVED** [review]

**Comment by jimmygchen** [comment]
@mergify queue

**Comment by jimmygchen** [comment]
@mergify requeue

**Comment by jimmygchen** [comment]
@mergify requeue

**Comment by jimmygchen** [comment]
@mergify requeue

**Pull request merged**